src/sentry/static/sentry/app/views/settings/components/forms/field/fieldWrapper.tsx

```diff
@@ -3,7 +3,11 @@ import styled from '@emotion/styled';
 
 import space from 'app/styles/space';
 
-type Props = {
+/**
+ * Using Parameters<typeof FieldWrapper> in the Field component somehow
```

Comment:
Hmm would `React.ComponentProps` not work?

Reply:
@billyvg Does not, it gives me the same `TS2589: Type instantiation is excessively deep and possibly infinite.` error

Reply:
What about `FieldWrapper['props']`?

Reply:
@markstory Didn't work. I don't think you can do that with functional components.